pkg/bgpv1/manager/reconciler/service_test.go

```diff
@@ -628,9 +628,10 @@ func TestLBServiceReconciler(t *testing.T) {
 			}
 
 			newc := &v2alpha1api.CiliumBGPVirtualRouter{
-				LocalASN:        64125,
-				Neighbors:       []v2alpha1api.CiliumBGPNeighbor{},
-				ServiceSelector: tt.newServiceSelector,
+				LocalASN:              64125,
+				Neighbors:             []v2alpha1api.CiliumBGPNeighbor{},
+				ServiceSelector:       tt.newServiceSelector,
+				ServiceAdvertisements: []v2alpha1api.BGPServiceAddressType{v2alpha1api.BGPLoadBalancerIPAddr},
```

Comment:
Could you please extend this test / add a new test to validate `ClusterIP` advertisements as well?

Similarly, could you please extend the tests in this file?https://github.com/cilium/cilium/blob/4082bc38407a3fd2b59f2bd1e8e0e1ed06be8340/pkg/bgpv1/test/adverts_test.go#L438